lib/libc/tests/gen/posix_spawn/Makefile

```diff
@@ -19,8 +19,12 @@ CLEANFILES+=	h_nonexec
 
 .include "../../Makefile.netbsd-tests"
 
+# The dd status=none option is non-standard. Only use it when this test succeeds
+# rather than require dd to be a bootstrap tool.
+DD_NOSTATUS!=(dd status=none count=0 2> /dev/null && echo status=none) || true
+DD=dd ${DD_NOSTATUS}
```

Comment:
I'm sure this will get pushback upstream, but IMO it's fine.

Reply:
Copied from https://reviews.freebsd.org/rS309412

Reply:
It might make sense to centralize the definition of DD_NOSTATUS in some bsd.foo.mk and then fix the 3 places to just s/status=none/DD_NOSTATUS/.

Reply:
The one reason not to is avoid running dd all the time.

Reply:
Yeah I wish we had a `configure` step where we could detect stuff like this once rather then in every makefile.